### PR TITLE
Suicide Vest Traitor Item

### PR DESCRIPTION
**Pull request by ToasterBiome**
# Document the changes in your pull request

Adds an extremely obvious suicide vest to the game that traitors can buy for 7 TC. It functions like a voice activated bomb, using an inclusive voice recorder. It makes a 1, 2, 4 explosion (same as syndicate minibomb) when it explodes, and you can only set the phrase one time (better not mess it up) additionally it's cursed so it can never be removed. When you die it also doesnt explode, so you best be quick.

# Why is this good for the game?

# Testing
image
image

# Spriting
image

# Changelog

:cl:
rscadd: Adds a suicide vest for traitors to buy
/:cl:

### CODE REVIEW

**Review by Mqiib — CHANGES_REQUESTED** [review]
Make it something other than the reactive armor sprite

**Comment by ToasterBiome** [comment]
> Make it something other than the reactive armor sprite

got any ideas?

**Comment by ShoziOffical** [comment]
Why do this when you can just activate a bomb in my hand

**Comment by iloveloopers** [comment]
What a waste of 7 TC, you can just make a normal voice activated atmos bomb

**Comment by wonderinghost** [comment]
looper i swear im going to add toolset implants to the uplink

**Comment by iloveloopers** [comment]
> looper i swear im going to add toolset implants to the uplink

That would actually be useful as long as it isn't super expensive

Making atmos bombs in maints is quite easy.

**Comment by ToasterBiome** [comment]
> What a waste of 7 TC, you can just make a normal voice activated atmos bomb

save yourself the 7 tc and make that instead

TC acts partially as a resource to get things you're unable to get, and partially as time gated things that you can get quicker (such as sentience potions)

**Comment by LassiH** [comment]
the wheelchair bomb already exists which is cheaper and has a bigger explosion

**Comment by Mqiib** [comment]
> > Make it something other than the reactive armor sprite
> 
> got any ideas?

Ideally something that looks like a bomb vest. Can't be that hard, it's not that big of a sprite. Perhaps along the lines of:

image

Basically a bunch of C4 and some wires strapped to a regular armor vest or something

**Comment by ToasterBiome** [comment]
I'll cook something up soon

**Comment by ynot01** [comment]
> > Make it something other than the reactive armor sprite
> 
> 
> 
> got any ideas?

theres literally a bomb vest sprite in the files already, the experimentor uses it i think

**Review by Mqiib — APPROVED** [review]
cool

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -598,3 +598,69 @@
 	icon_state = "pocketcat"
 	item_state = "pocketcat"
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS
+
+/obj/item/clothing/suit/unalivevest
+	name = "extremely obvious suicide vest"
```
Comment (ynot01) [inline code/modules/clothing/suits/miscellaneous.dm]: this name is lame

File: code/modules/clothing/suits/miscellaneous.dm [hunk 2]
```diff
@@ -598,3 +598,69 @@
...
+	desc = "An autolocking, voice activated suicide vest. The electronics inside are so crude it only functions in inclusive mode. Once it's on, it can never be removed."
+	icon = 'icons/obj/clothing/suits/suits.dmi'
+	icon_state = "reactiveoff"
```
Comment (ynot01) [inline code/modules/clothing/suits/miscellaneous.dm]: this is not the sprite you made

**Review by ynot01 — CHANGES_REQUESTED** [review]

File: code/modules/clothing/suits/miscellaneous.dm (hunk 2 above)
Comment (ToasterBiome) [reply code/modules/clothing/suits/miscellaneous.dm]: fixed

File: code/modules/clothing/suits/miscellaneous.dm (hunk 1 above)
Comment (ToasterBiome) [reply code/modules/clothing/suits/miscellaneous.dm]: what should it be instead

Comment (ynot01) [reply code/modules/clothing/suits/miscellaneous.dm]: explosive vest would be better imo

**Review by ynot01 — APPROVED** [review]

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -598,3 +598,68 @@
 	icon_state = "pocketcat"
 	item_state = "pocketcat"
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS
+
+/obj/item/clothing/suit/unalivevest
+	name = "extremely obvious suicide vest"
```
Comment (ToasterBiome) [inline code/modules/clothing/suits/miscellaneous.dm]:
```suggestion
	name = "explosive vest"
```

**Pull request merged**